client/src/ui/track/JobCard.tsx

```diff
@@ -245,13 +261,36 @@ export function JobCard({ application, profile }: JobCardProps) {
               padding: "6px",
               borderRadius: "50%",
             }}
-            onClick={(e) => e.stopPropagation()}
+            onClick={(e) => {
+              e.stopPropagation()
+              if (link) {
+                navigator.clipboard.writeText(link)
+                setAlert({
+                  message: "Link copied to clipboard",
+                  type: "success",
+                })
+              } else {
+                setAlert({
+                  message: "No link available",
+                  type: "error",
+                })
+              }
+            }}
           >
             <FontAwesomeIcon
               icon={faLink}
               style={{ fontSize: "14px", color: "#333333" }}
             />
           </IconButton>
+          <IconButton
```

Comment:
nit - I want to keep the bottom 3 icons grouped up for information rather than utility. What I suggest is to have a utility button (3 bar) where it renders the modal with options when clicked. Ex:
![image](https://github.com/user-attachments/assets/51186b15-4e10-45f8-b6ba-765a082fdcde)

Reply:
To be honest, I prefer display it on the job card better. It makes our platform stand out. The 3 bar would work also but having 3 icons group under it make the job card looks really bland which can leave it too many step for user to interact. So I would keep it visible to user. And make the user experience much more fun, plus I already did the logic for each button and those communicate well between the modal.

Reply:
Sounds good. If you're going to keep it that way, can we remove the triple bar icon then since it has no function

Reply:
okie